pg_hint_plan.c

```diff
@@ -687,6 +687,8 @@ _PG_init(void)
 							 NULL,
 							 NULL);
 
+	EmitWarningsOnPlaceholders("pg_hint_plan");
```

Comment:
This is a good idea as we may add new parameters in stable branches, so applied down to 10 separately.